lib/synced/strategies/full.rb

```diff
@@ -93,6 +90,10 @@ def perform
         end
       end
 
+      def reset_synced
+        # this is not supported by this strategy
```

Comment:
hmmm, reset_synced just don't make sense for full strategy, its pointless to call this for full strategy. I added it here so that strategy interface is uniform. It does no harm to call this method, not convinced we should really blow on this, @Azdaroth @ZenCocoon ? @Fredar what should be the error then? `NotImplementedError` ?